MeetU/MeetU/Scripts/Controllers/Meetup/edit.js

```diff
@@ -0,0 +1,48 @@
+﻿(function () {
+    "use strict";
+    angular
+        .module('meetupModule')
+        .controller('MeetupEditController', MeetupEditController)
+    MeetupEditController.$inject = ["$rootScope", "$log", "$resource", "$location", "$scope", "$timeout"];
+    function MeetupEditController($rootScope, $log, $resource, $location, $scope, $timeout) {
+        var vm = this;
+        $rootScope.$on("UpdateMeetup", function (event, message) {
```

Comment:
it should go like this:
1.user click edit button
2.in index, pass a meetup value to service
3.navigate to edit page
4.in edit page, get meetup value from service
5.user fills in values
6.user click ok button
7.if ok, save everything to server, broadcast a success event. in index, receive success event and update array. then navigate back to index